build.gradle

```diff
@@ -68,6 +69,10 @@ subprojects {
         }
     }
 
+    configurations.all {
+        exclude group: 'ch.qos.logback', module: 'logback-classic'
```

Comment:
you probably want all shaded dependencies here and explicitly filter mantis-shaded here.